src/amber/controller/base.cr

```diff
@@ -15,13 +15,15 @@ module Amber::Controller
     protected getter params : Amber::Validators::Params
     protected getter flash : Amber::Router::Flash::Params
     protected getter session : Amber::Router::Session::Params
+    protected getter cookies : Amber::Router::Cookies::Store?
 
     def initialize(@context : HTTP::Server::Context)
       @request = context.request
       @response = context.response
       @raw_params = context.params
       @flash = context.flash
       @session = context.session
+      @cookies = context.cookies
```

Comment:
Lets not copy cookies to controller. We already have session and flash. If they want to manipulate other cookies they can call `context.cookies`.